test/api-e2e/api_test.go

```diff
@@ -59,8 +61,21 @@ func (s *APISuite) TestISBSVC() {
 	getISBSVCBody := HTTPExpect(s.T(), "https://localhost:8443").GET(fmt.Sprintf("/api/v1/namespaces/%s/isb-services/%s", Namespace, testISBSVCName)).
 		Expect().
 		Status(200).Body().Raw()
+	for !strings.Contains(getISBSVCBody, `"status":"healthy"`) {
+		select {
+		case <-ctx.Done():
+			if ctx.Err() == context.DeadlineExceeded {
+				s.T().Fatalf("failed to get namespaces/isb-services: %v", ctx.Err())
+			}
+		default:
+			time.Sleep(100 * time.Millisecond)
+			getISBSVCBody = HTTPExpect(s.T(), "https://localhost:8443").GET(fmt.Sprintf("/api/v1/namespaces/%s/isb-services/%s", Namespace, testISBSVCName)).
+				Expect().
+				Status(200).Body().Raw()
+		}
+	}
 	assert.Contains(s.T(), getISBSVCBody, fmt.Sprintf(`"name":"%s"`, testISBSVCName))
-	assert.Contains(s.T(), getISBSVCBody, `"status":"inactive"`)
+	assert.Contains(s.T(), getISBSVCBody, `"status":"healthy"`)
```

Comment:
Do we still need this line?

Reply:
Not anymore since we are waiting for ISBSvc to be healthy already, removing.